tensorly/contrib/decomposition/tt_TTOI.py

```diff
@@ -0,0 +1,173 @@
+
+import numpy as np
+from math import prod
+import tensorly as tl
+from tensorly.tt_tensor import tt_to_tensor, validate_tt_rank
+import warnings
+
+
+#Author: Lili Zheng
+
+#Reference paper: Zhou Y, Zhang AR, Zheng L, Wang Y. "Optimal high-order tensor svd via tensor-train orthogonal iteration."
```

Comment:
It's better to move the reference to the docstrings as we did in other algorithms so you can refer to them and they appear in the documentation and when users check `help`

Reply:
Moved the reference to the docstring now.